glossary.html

```diff
@@ -3,6 +3,8 @@
 <head>
     <link rel="stylesheet" href="stylesheet/normalize.css" type="text/css">
     <link rel="stylesheet" href="stylesheet/main.css" type="text/css">
+    <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
```

Comment:
Incorrect path in href
You didn't go through the favicon generator process.  You should be using a favicon.ico file rather than a plain png